server/go.mod

```diff
@@ -2,7 +2,7 @@ module github.com/mattermost/mattermost/server/v8
 
 go 1.23.0
 
-toolchain go1.23.7
+toolchain go1.23.9
 
 //replace github.com/mattermost/mattermost/server/public => /Users/ibrahim/go/src/github.com/mattermost/mattermost-server/server/public
```

Comment:
Oh no .. @isacikgoz .. Probably snuck in from your ABAC PR :stuck_out_tongue:

Reply:
I pushed a commit to remove the comment.